.circleci/config.yml

```diff
@@ -2,15 +2,15 @@ version: 2
 jobs:
   build:
     docker: &docker
-      - image: ruby:2.5.1-alpine
+      - image: ruby:2.2.10-alpine
```

Comment:
Currently support ruby-2.2 because I don't want to use heredoc style `<<~` yet.

I will decide the supported ruby versions in the near future. It related to #70.